tools/requires_log_to_webtreemap.rb

```diff
@@ -0,0 +1,49 @@
+if ARGV.length != 2
+  puts "Usage: #{$0} /path/to/requires.log /path/to/webtreemap"
+  puts
+  puts "This tool uses webtreemap, which can be obtained from https://github.com/evmar/webtreemap"
+  exit 1
```

Comment:
ugh, this exit is causing the coverage helper to fail because you can't require this file.